flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -400,6 +400,9 @@ static void uavoFrSKYSensorHubBridgeTask(void *parameters)
 			// As long as there is no voltage for each cell
 			// all cells will have the same voltage.
 			// Receiver will know number of cells.
+			if(batState.DetectedCellCount)
+				batSettings.NbCells = batState.DetectedCellCount;
```

Comment:
Not sure about this one , it might interfere with the logic above https://github.com/henn1001/dRonin/blob/CellDetection/flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c#L268